nrf_security/doc/driver_config.rst

```diff
@@ -253,12 +299,41 @@ You can use the following Kconfig options for fine-grained control over which dr
 +-----------------------+--------------------------------------------------------------------------+---------------------------------------------------------------------------+
-| secp384r1             | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ECC_SECP_R1_384_CC3XX`         | Not supported                                                             |
+| secp384r1             | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ECC_SECP_R1_384_CC3XX`         | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ECC_SECP_R1_384_OBERON`         |
 +-----------------------+--------------------------------------------------------------------------+---------------------------------------------------------------------------+
-| secp521r1             | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ECC_SECP_R1_521_CC3XX`         | Not supported                                                             |
+| secp521r1             | Not supported                                                            | Not supported                                                             |
 +-----------------------+--------------------------------------------------------------------------+---------------------------------------------------------------------------+
 
 
+RNG configurations
+******************
+
+To enable PRNG seeded by entropy (also known as TRNG), set one or more of the following configurations:
+
++---------------------------+-------------------------------------------------+
+| PRNG algorithms           | Configuration option                            |
++===========================+=================================================+
+| CTR_DRBG                  | :kconfig:option:`CONFIG_PSA_WANT_ALG_CTR_DRBG`  |
++---------------------------+-------------------------------------------------+
+| HMAC_DRBG                 | :kconfig:option:`CONFIG_PSA_WANT_ALG_HMAC_DRBG` |
++---------------------------+-------------------------------------------------+
+
+.. note::
+   * Both PRNG algorithms are NIST qualified Cryptographically Secure Pseudo Random Number Generators (CSPRNG).
+   * :kconfig:option:`CONFIG_PSA_WANT_ALG_CTR_DRBG` and :kconfig:option:`CONFIG_PSA_WANT_ALG_HMAC_DRBG` are custom configurations not described by the PSA Crypto specification.
+   * If multiple PRNG algorithms are enabled at the same time, CTR_DRBG will be priortized for random number generation through the frontend APIs for PSA crypto.
+
+
+RNG driver configurations
+*************************
+
+There aren't any public configurations for entropy and PRNG algorithm support and the choice of drivers that provide support is automatic.
+
+The PSA drivers using the Arm CryptoCell peripheral is enabled by default for nRF52840, nRF91 series, and nRF5340 devices.
+
+For devices without a hardware accelerated cryptographic engine, entropy is provided from nRF RNG periperal and PRNG support is provided by Oberon PSA driver implemented using software.
```

Comment:
```suggestion
For devices without a hardware-accelerated cryptographic engine, entropy is provided by the nRF RNG peripheral. PRNG support is provided by the Oberon PSA driver implemented using software.
```

I'm not really sure how to parse the 'driver implemented using software'. Is the driver implemented through software?

Reply:
Adjusted the text according to your proposals

With regards to your comment on "driver implementing software":
PRNG could be provided either by hardware or by software. The PSA Crypto drivers are normally written to give access to hardware accelerated cryptography instead of builtin software in Mbed TLS deliverable. the Oberon PSA crypto driver is implemented in software, which bears mentioning.

Reply:
OK, I adjusted my suggestion to make the clause structure a little clearer.

Reply:
Updated to your latest proposal, which is much clearer!

Fixed